styles/callback-bridge.css

```
@@ -0,0 +1,93 @@
+  stroke-linejoin: round;
+}
+
+.status-error .status-text {
+  color: #fca5a5;
+}
+
+.error-detail {
+  font-size: 0.75rem;
+  color: #64748b;
+  margin-top: 4px;
+  word-break: break-word;
```

Stylelint (17.7.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)